specifyweb/frontend/js_src/lib/components/QueryBuilder/Field.tsx

```diff
@@ -233,6 +233,9 @@ export function QueryLine({
 
   const hasAny = field.filters.some(({ type }) => type === 'any');
 
+  const lightGrayButton = `${className.grayButton} hover:!bg-gray-400 !bg-gray-100 !text-gray-800 !shadow-sm
```

Comment:
Good job customizing all the other styles too!
Could you please move this line into `./components/Atoms/className.ts` right after the `grayButton`?

https://github.com/specify/specify7/blob/1f3adff00d01d09a3b572951f68bb52d14e2de9e/specifyweb/frontend/js_src/lib/components/Atoms/className.ts#L54

And then you can use it in the code as `className.lightGrayButton`

Also, it looks like you ended up overriding all the styles of the `className.grayButton`, so instead of overriding them, you could just start from scratch (i.e, copy the value of `grayButton` for `lightGrayButton`, but change the styles that differ, instead of using `!` to override them)

Reply:
Thanks - I thought you might want it there :smiley: Got it changed over